util/fhir.ts

```diff
@@ -34,6 +35,36 @@ export function createPatientResourceString(birthDate: string): string {
   return JSON.stringify(pt, null, 2);
 }
 
+/**
+ * Identifies the primary code path of a resource and constructs a string which displays
+ * resource summary information depending on what is a available
+ * @param resource {Object} a fhir Resource object
+ * @returns {String} displaying the code and display text, code, or id of the resource or nothing
+ */
+export function getFhirResourceSummary(resource: fhir4.Resource) {
+  const primaryCodePath = parsedPrimaryCodePaths[resource.resourceType]?.primaryCodePath;
+  const resourceCode = `${fhirpath.evaluate(resource, `${primaryCodePath}.coding.code`)}`;
+  const resourceDisplay = `${fhirpath.evaluate(resource, `${primaryCodePath}.coding.display`)}`;
+
+  if (primaryCodePath) {
+    if (resourceCode !== '' && resourceDisplay !== '') {
```

Comment:
```suggestion
  if (primaryCodePath) {
      const resourceCode = `${fhirpath.evaluate(resource, `${primaryCodePath}.coding.code`)}`;
      const resourceDisplay = `${fhirpath.evaluate(resource, `${primaryCodePath}.coding.display`)}`;
    if (resourceCode !== '' && resourceDisplay !== '') {
```

Suggest moving the code and display grabbing to inside this if so it is only executed if we do have a `primaryCodePath`.